app/src/androidTest/java/com/github/polypoly/app/commons/MockDB.kt

```diff
@@ -71,6 +73,28 @@ class MockDB: IRemoteStorage {
         return CompletableFuture.completedFuture(true)
     }
 
+    override fun removeValue(key: String): CompletableFuture<Boolean> {
+        TODO("Not yet implemented")
+    }
+
+    override fun <T : Any> addOnChangeListener(
+        key: String,
+        tag: String,
```

Comment:
What's the point of the tag? It doesn't seem used in this PR (maybe I missed something)

Reply:
So far it is only used in tests, however it is pretty useful as some DB elements may be listened by different activities for example.